### PR TITLE
Report elements

### PR DESCRIPTION
**Pull request by gislenius**

### CODE REVIEW

File: swagger/queries.json [hunk 1]
```diff
@@ -2467,6 +2467,20 @@
                 "type": "string",
```
Comment (wcandillon) [inline swagger/queries.json]: @gislenius The disclosure parameter is still documented as sec only.

File: queries/public/api/labels.jq [hunk 2]
```diff
@@ -27,6 +27,9 @@ declare  %rest:case-insensitive %rest:distinct  %rest:encoding("UTF-8") variable
 declare  %rest:case-insensitive %rest:distinct  variable $language           as string* external;
 declare  %rest:case-insensitive %rest:distinct  variable $labelRole          as string* external;
 declare  %rest:case-insensitive                 variable $profile-name       as string  external := $config:profile-name;
+declare  %rest:case-insensitive                 variable $onlyTextBlocks     as boolean? external := ();
+declare  %rest:case-insensitive                 variable $kind               as string*  external := ();
```
Comment (wcandillon) [inline queries/public/api/labels.jq]: @gislenius The annotation says case insensitive but the parameter doesn't seem to be case insensitive

File: queries/public/api/labels.jq [hunk 3]
```diff
@@ -27,6 +27,9 @@ declare  %rest:case-insensitive %rest:distinct  %rest:encoding("UTF-8") variable
...
+declare  %rest:case-insensitive                 variable $eliminateReportElementDuplicates as boolean external := false;
```
Comment (wcandillon) [inline queries/public/api/labels.jq]: @gislenius It didn't seem to work.

File: swagger/queries.json (hunk 1 above)
Comment (gislenius) [inline swagger/queries.json]: DONE

File: queries/public/api/labels.jq (hunk 2 above)
Comment (gislenius) [inline queries/public/api/labels.jq]: DONE

File: queries/public/api/labels.jq (hunk 3 above)
Comment (gislenius) [inline queries/public/api/labels.jq]: I checked that it works

**Pull request merged**